internal/server/stat/golden/get_stat_all_test.go

```diff
@@ -33,10 +33,7 @@ func TestGetStatAll(t *testing.T) {
 	_, filename, _, _ := runtime.Caller(0)
 	goldenPath := path.Join(path.Dir(filename), "get_stat_all")
 
-	for _, opt := range []*e2e.TestOption{
-		{},
-		{UseImportGroup: true},
-	} {
+	testSuite := func(opt *e2e.TestOption) {
```

Comment:
when running this as performance test, should we stop before all the golden read and comparison. So the latency is purely for the API calls